LICENSE

```diff
@@ -0,0 +1,9 @@
+Copyright 2017 Idealnaya rabota LLC (Republic of Belarus)
```

Comment:
Do you have any third-party dependencies and\or submodules ? If yes, then please include extra clause about licenses of those.